ufo/__init__.py

```diff
@@ -26,30 +25,17 @@
 db = sqlalchemy.SQLAlchemy(app)
 
 # DB needs to be defined before this point
+import database
 import models
-
-@app.after_request
-def commitDB(response):
-  """Save credentials if changed, then commit the db session"""
-  # credentials are cached in the app context, save them if they have changed
-  credentials = getattr(flask.g, '_credentials', None)
-  if credentials is not None:
-    config = get_user_config()
-    json_credentials = credentials.to_json()
-    if config.credentials != json_credentials:
-      config.credentials = json_credentials
-
-  db.session.commit()
-
-  return response
+from models import Config
```

Comment:
Do you need this specifically since models is already imported overall? If we're not referencing models anymore, then maybe remove that and only have this import.